GooglePlayInstant/Samples/TestApp/Scripts/TestApp.cs

```diff
@@ -24,6 +27,30 @@ public class TestApp : MonoBehaviour
         private const string CookiePrefix = "test-cookie";
         private string _storedCookie;
 
+        private Dictionary<KeyCode, Action> _keyMappings;

```

Comment:
It's not letting me comment on unmodified lines. Anyway for the class documentation:

"Tests instant app plugin features through button clicks"

I'd change to something like "Tests instant app plugin features via buttons for manual testing and key presses for automated testing."

Reply:
Great catch